### DISCUSSION TITLE
What definition schould I enable for the EHBX08E9W and ERGA08EVH7

### DISCUSSION THREAD

**Discussion by G0b3** [raomin/ESPAltherma#333] [Q&A]
I'm struggling which definition i should use. I tried to search on the forum for the same model but I can't find it. 

Maybe the 
def/ALTHERMA(LT_DA_04-08KW)
or just 
def/DEFAULT

Suggestions?

**Comment by Bolo92** [comment]
I have EHBH08DA9W (indoor) and ERGA08DAV3 (outdoor) and i choosed def\ALTHERMA(LT_CA_CB_04-08KW).
I compared side to side few files. The registers are the same (or almost). In LT_CA_CB_04-08KW is more registers than in DEFAULT like flow water meter ( liters / minute) or water pressure and few others.

If you compare LT_CA_CB_04-08KW vs HYBRID is almost the same.
